docs/source/gbd2019_models/causes/ihd/index.rst

```diff
@@ -41,9 +41,91 @@ Chronic ischemic heart disease can cause signs and symptoms such as angina, anxi
+  c. Sudden (abrupt) unexplained cardiac death, involving cardiac arrest or no evidence of a non-coronary cause of death
+  d. Prevalent MI is considered to last from the onset of the event to 28 days after the event and is divided into an acute phase (0-2 days) and subacute (3-28 days).
+
+2. Chronic IHD:
+  
+  a. Angina; clinically diagnosed stable exertional angina pectoris or definite angina pectoris according to the Rose Angina Questionnaire, physician diagnosis, or taking nitrate medication for the relief of chest pain.
+  b. Asymptomatic ischaemic heart disease following myocardial infarction; survival to 28 days following incident MI. The GBD study does not use estimates based on ECG evidence for prior MI, due to its limited specificity and sensitivity.
+
+
+**Input data:**\
+
+Apart from inpatient hospital and inpatient claims data, we did not include any data from sources other than the literature for myocardial infarction. 
+
+The primary input for the asymptomatic ischaemic heart disease following myocardial infarction model are 28-day survivors calculated from the excess mortality estimates for the myocardial infarction model. We included data for excess mortality and standardised mortality ratio to inform the estimates of survival after myocardial infarction.
+
+For angina, the data sources used include literature data, survey data, and U.S. claims data (but we did not include inpatient hospital data from any locations). All outpatient data were excluded as they were implausibly low for all locations when compared with literature and claims data.
+
+**Severity split inputs:**\
+
+Acute myocardial infarction was split into two severity levels by length of time since the event – days 1 and 2 versus days 3 through 28. Disability weights were established for these two severities using the standard approach for GBD 2019. 
+
+.. list-table:: Severity distribution for acute myocardial infarction
+   :widths: 15 25 12
+   :header-rows: 1
+
+   * - Severity level
+     - Lay description
+     - DW (95% CI)
+   * - Acute myocardial infarction, days 1-2
+     - Has severe chest pain that becomes worse with any physical activity. The person feels nauseated, short of breath, and very anxious
+     - 0.432 (0.288–0.579)
+   * - Acute myocardial infarction, days 3-28 
+     - Gets short of breath after heavy physical activity, and tires easily, but has no problems when at rest. The person has to take medication every day and has some anxiety. 
+     - 0.074 (0.049–0.105)
+
+Asymptomatic ischaemic heart disease following myocardial infarction was all assigned to the asymptomatic severity level. No disability weight is assigned to this level. 
+
+.. list-table:: Severity distribution for asymptomatic ischaemic heart disease following myocardial infarction
+   :widths: 15 25 12
+   :header-rows: 1
+
+   * - Severity level
+     - Lay description
+     - DW (95% CI)
+   * - Asymptomatic ischaemic heart disease
+     - 
+     - N/A
```

Comment:
It is probably appropriate to have DW 0.0 for Asymptomatic IHD, and that will be easier for the engineers to interpret than N/A.

Reply:
That makes sense! I have updated to change N/A to 0.0 for the disability weights.